department-of-veterans-affairs/va.gov-cms
# VACMS-12549: Update focus state on admin tool bar in CMS

## Description

Closes #12549.

## Testing done
Tested locally, focus state for is updated and breadcrumb now has green border for focus. 

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

1. Check admin toolbar
   - [ ] Validate that focus state is fixed per AC
2. Then check focss state for breadcrumb
   - [ ] Validate that fix matches AC

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
